### PR TITLE
sdl2_net: new package at version 2.0.1

### PR DESCRIPTION
**Pull request by tslilc**
Please fill out the template.

Read: https://k1sslinux.org/wiki/kiss/style-guide

Please include the manifest for version bumps and new packages.

---

## Description of package

From upstream [0]:

> This is a small sample cross-platform networking library, with a sample chat client and server application. The chat client uses the [GUIlib](https://www.libsdl.org/projects/GUIlib/index.html) GUI framework library.

[0] [https://www.libsdl.org/projects/SDL_net/](https://www.libsdl.org/projects/SDL_net/)

## Installed manifest of package

(`kiss manifest <pkg>`)

```
/var/db/kiss/installed/sdl2_net/version
/var/db/kiss/installed/sdl2_net/sources
/var/db/kiss/installed/sdl2_net/manifest
/var/db/kiss/installed/sdl2_net/depends
/var/db/kiss/installed/sdl2_net/checksums
/var/db/kiss/installed/sdl2_net/build
/var/db/kiss/installed/sdl2_net/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/lib/pkgconfig/SDL2_net.pc
/usr/lib/pkgconfig/
/usr/lib/libSDL2_net.so
/usr/lib/libSDL2_net.a
/usr/lib/libSDL2_net-2.0.so.0.0.1
/usr/lib/libSDL2_net-2.0.so.0
/usr/lib/
/usr/include/SDL2/SDL_net.h
/usr/include/SDL2/
/usr/include/
/usr/
```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

## Existing package

- [ ] I am the maintainer of this package.

### CODE REVIEW

File: community/sdl2_net/build [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
+
+./configure --prefix=/usr
```
Comment (aabacchus) [inline community/sdl2_net/build:3]: It is the preferred style to split this:
```sh
./configure \
    --prefix=/usr
````
https://kisslinux.org/wiki/package-system#001

Comment (tslilc) [reply community/sdl2_net/build]: Thanks!

**Comment by git-bruh** [comment]
Doesn't seem to require libtool at all

**Comment by tslilc** [comment]
Thanks, i've removed the dependency. My apologies

**Pull request merged**